File: mobile/iOS/GuestJini/GuestJini/Global/Model/Responses/UserInterestsResponse.swift
//
//  UserInterestsResponse.swift
//  GuestJini
//
//  Created by Prasanna Kumar Pete on 23/01/20.
//  Copyright © 2020 Prasanna Kumar Pete. All rights reserved.
//

import Foundation
class UserInterestsResponse: Decodable {
    public var userInterests:UserInterests? = UserInterests()
    public var userInterestsList:[UserInterests]?  = []
    public var totalPages:Int? = 0
    public var totalRecords:Int? = 0
    public var currentRecords:Int? = 0
    public var success:Bool? = false
    public var error:String? = ""
}

File: mobile/android/GuestJini/app/src/main/java/com/carmel/guestjini/Networking/Users/UserInterestsResponse.java
package com.carmel.guestjini.Networking.Users;

import java.util.List;

public class UserInterestsResponse {
    private UserInterests userInterests;
    private List<UserInterests> userInterestsList;
    private int totalPages;
    private int totalRecords;
    private int currentRecords;
    private boolean success;
    private String error;

    public UserInterests getUserInterests() {
        return userInterests;
    }

    public void setUserInterests(UserInterests userInterests) {
        this.userInterests = userInterests;
    }

    public List<UserInterests> getUserInterestsList() {
        return userInterestsList;
    }

    public void setUserInterestsList(List<UserInterests> userInterestsList) {
        this.userInterestsList = userInterestsList;
    }

    public int getTotalPages() {
        return totalPages;
    }

    public void setTotalPages(int totalPages) {
        this.totalPages = totalPages;
    }

    public int getTotalRecords() {
        return totalRecords;
    }

    public void setTotalRecords(int totalRecords) {
        this.totalRecords = totalRecords;
    }

    public int getCurrentRecords() {
        return currentRecords;
    }

    public void setCurrentRecords(int currentRecords) {
        this.currentRecords = currentRecords;
    }

    public boolean isSuccess() {
        return success;
    }

    public void setSuccess(boolean success) {
        this.success = success;
    }

    public String getError() {
        return error;
    }

    public void setError(String error) {
        this.error = error;
    }
}
